python-api/src/main/python/api/__init__.py

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
Basically you're calling the Livy python API module "api", which is too generic. It should be called something else, e.g., `livy`.